### DISCUSSION TITLE
Group DM in v2 API

### DISCUSSION THREAD

**Discussion by crepusculi** [TwidereProject/TwidereX-Android#240] [General] [answered]
I've looked everywhere, and i was wondering if you had any idea when or even if Twitter was planning on implementing group DMs in the v2 API. I wouldn't expect it to be soon, but I wasn't sure if you were privy to anything since you have been approved for v2 usage.

**Comment by Tlaster** [comment, marked as answer]
We also have no idea when Twitter will release the full V2 API ATM, and we're still waiting for Twitter to release more V2 APIs.
